chartfx-dataset/src/test/java/de/gsi/dataset/spi/DataSetEqualityTests.java

```diff
@@ -60,7 +61,14 @@ public void testDataSetEquality() {
                                 new double[] { 9, 8, 7 } }),
                 new DoubleDataSet3D("test", new double[] { 1, 2, 3 }, new double[] { 6, 7, 8 }, new double[][] {
                         new double[] { 1, 2, 3 }, new double[] { 6, 5, 4 }, new double[] { 9, 8, 7 } }));
-
+        assertEquals(new DimReductionDataSet(
+                new DoubleDataSet3D("test", new double[] { 1, 2, 3 }, new double[] { 6, 7, 8 },
+                        new double[][] { new double[] { 1, 2, 3 }, new double[] { 6, 5, 4 },
+                                new double[] { 9, 8, 7 } }), DataSet.DIM_X, Option.SLICE),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'DataSet.DIM_X' due to existing static import 'de.gsi.dataset.DataSet.DIM_X'](https://app.codacy.com/app/GSI/chart-fx/pullRequest?prid=4728036)